lib/hu_board/services/slack.rb

```diff
@@ -29,7 +29,7 @@ def github_url
     def transform_moved(mash)
       {
         username: "#{mash.meta.user.login} via HuBoard",
-        icon_url: "https://avatars.githubusercontent.com/u/#{mash.meta.user.id}?s=64",
+        icon_url: "#{mash.payload.issue.user.avatar_url}s=64",
```

Comment:
That's a good point, custom enterprise image servers might not be tacking API versioning onto their URL's. I didn't think of that